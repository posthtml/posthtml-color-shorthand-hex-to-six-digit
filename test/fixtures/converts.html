<!DOCTYPE html>
<html>
<head>
  <style type="text/css">
    @media (max-width: 600px) {
      .real-class-1#head-only-class-1[lang|en]{width:100% !important;}
      #real-id-1.head-only-class-1:hover{display: block !important;}
      .head-only-class-2[lang|en]{color: #ccc !important;}
      @media (max-width: 200px) {
        #real-id-1{background-color: #000;}
      }
      @media (max-width: 100px) {
        .head-only-class-1{border: 1px solid #FFF !important;}
      }
    }
  </style>
  <title>zzzz</title>
  <style type="text/css">
    .real-class-1#head-only-class-1[lang|en]{color: #CC00FF !important;}
    #real-id-1.head-only-class-1:hover{display: block !important;}
    .head-only-class-3[lang|en]{background-color: #ff0 !important;}
    div .real-class-1 a:hover {color: #0000cc;}
  </style>
</head>
<body>
  <table id="real-id-1" width="100%" border="0" cellpadding="0" cellspacing="0">
    <tr>
      <td class="real-class-1" style="color: #FFc;">
        <img src="spacer.gif" alt="spacer">
      </td>
    </tr>
  </table>
</body>
</html>
