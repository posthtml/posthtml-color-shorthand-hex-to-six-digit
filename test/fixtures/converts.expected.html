<!DOCTYPE html>
<html>
<head>
  <style type="text/css">
    @media (max-width: 600px) {
      .real-class-1#head-only-class-1[lang|en]{width:100% !important;}
      #real-id-1.head-only-class-1:hover{display: block !important;}
      .head-only-class-2[lang|en]{color: #cccccc !important;}
      @media (max-width: 200px) {
        #real-id-1{background-color: #000000;}
      }
      @media (max-width: 100px) {
        .head-only-class-1{border: 1px solid #ffffff !important;}
      }
    }
  </style>
  <title>zzzz</title>
  <style type="text/css">
    .real-class-1#head-only-class-1[lang|en]{color: #cc00ff !important;}
    #real-id-1.head-only-class-1:hover{display: block !important;}
    .head-only-class-3[lang|en]{background-color: #ffff00 !important;}
    div .real-class-1 a:hover {color: #0000cc;}
  </style>
</head>
<body>
  <table id="real-id-1" width="100%" border="0" cellpadding="0" cellspacing="0">
    <tr>
      <td class="real-class-1" style="color: #ffffcc;">
        <img src="spacer.gif" alt="spacer">
      </td>
    </tr>
  </table>
</body>
</html>
